### DISCUSSION TITLE
How to make ssr.js api for Vercel work with custom server

### DISCUSSION THREAD

**Discussion by ndavd** [vikejs/vike#1301] [Discussion] [answered]
Hey, I have made a small route in `server/index.ts` for an endpoint which I need in my project (https://github.com/ndavd/ncube/blob/7c5c850b943ffa5053c6c766dd0fe41efdbb2d91/web/server/index.ts#L48-L57). Unfortunately it doesn't work with the ssr.js API (https://github.com/ndavd/ncube/blob/7c5c850b943ffa5053c6c766dd0fe41efdbb2d91/web/api/ssr.js) that I need for the Vercel deployment

**Comment by brillout** [comment]
In my experience, Vercel's runtime has some discrepancies with Node.js.

**Reply by ndavd** [reply]
So I probably can't use `fetch`? All I'm trying to accomplish is a simple route that works in both. In the meantime I tried to manually do the route in ssr.js but it's still not working https://github.com/ndavd/ncube/blob/main/web/api/ssr.js — Vercel deploys but I get HTTP 500

**Reply by ndavd** [reply]
Maybe I can't import code from `../server`

**Reply by brillout** [reply]
I don't know, I'd need to dig into your code.

**Reply by ndavd** [reply, marked as answer]
Funny error @brillout turned out I was exceeding the hard 5MB limit of Vercel's LAMBDA functions
